@AH-64D Apache Longbow/Addons/fza_ah64_ase/functions/fn_aseManager.sqf

```diff
@@ -0,0 +1,50 @@
+/* ----------------------------------------------------------------------------
+Function: fza_ase_fnc_aseManager
+
+
+Description:
+    Handles all ASE systems on the aircraft
+
+Parameters:
+    heli: Object - Vehicle the event handler is assigned to
+
+Returns:
+    Nothing
+
+Examples:
+    [_heli] call fza_ase_fnc_aseManager
+
+Author:
+    Snow(Dryden)
+---------------------------------------------------------------------------- */
+#include "\fza_ah64_ase\headers\ase.h"
+params ["_heli"];
+
+if (!(player == driver _heli) && !(player == gunner _heli)) exitwith {};
+
+private _deltaTime = ["ase_deltaTime"] call BIS_fnc_deltaTime;
+private _dcBusOn   = _heli getVariable "fza_systems_dcBusOn";
+private _rlwrPwr   = _heli getVariable "fza_ah64_ase_rlwrPwr";
+private _aseUpdateTime = _heli getVariable "fza_ah64_ase_updateTimer";
+private _cmwsPwr   = (_heli animationPhase "msn_equip_british" == 1 && _heli getVariable "fza_ah64_ase_msnEquipPwr" == 1);
+
+if (_dcBusOn) then {
+    [_heli, _deltaTime] call fza_ase_fnc_irJam;
+    if ((_aseUpdateTime + (1 / 2)) - CBA_missionTime < 0) then {
+        _heli setVariable ["fza_ah64_ase_updateTimer", CBA_missionTime];
```

Comment:
This variable is declared publicly in the init and not here. You need to make sure to be consistent.

I say make it not public everywhere.

It may be worth you going through and ensuring others are consistent